.github/workflows/pycodestyle.yml

```diff
@@ -19,4 +19,4 @@ jobs:
           sudo python3 -m pip install --upgrade pip
           sudo python3 -m pip install pycodestyle
       - name: Run pycodestyle
-        run: pycodestyle snap7 tests
+        run: pycodestyle snap7 tests --ignore=E501,E402,W391,W292,W503
```

Comment:
shouldn't this be in the pycodestyle config?

Reply:
It should. But do we really need to add another config file?  Instead, how about keeping this parameter as a command-line option for now and then replacing `pycodestyle` with [ruff](https://github.com/astral-sh/ruff) that supports `pyproject.toml`? It will make project's configs more centralized. Ideally, transfer all configurations to `pyproject.toml`. [These](https://github.com/gijzelaerr/python-snap7/blob/b5ff3d7676a3ae0888d11b64278a65c8432ed905/.github/workflows/flake8.yml#L24) parameters too.

Reply:
whatever you prefer, i thought might have forgotten it. migrating to ruff is also fine with me.

Reply:
I'll take care of migrating to ruff in the next PR